as-clients/rfp.md

```diff
@@ -0,0 +1,93 @@
+Atlantean Banking: Request for Proposals
+========================================
+
+## Document History
+
+| Version | When            | Who                    | What                                      |
+| ---     | ---             | ---                    | ---                                       |
+| v1.0    | 2023-01-24      | Client team            | Initial drafting                          |
+
+## Table of Contents
+
+(1.0) Product Overview  
+(2.0) Project Objectives  
+(3.0) Current System(s)  
+(4.0) Intended Users of the System  
+(5.0) Known interactions with systems within or outside the client organization  
+(6.0) Known constraints to development  
+(7.0) Project Schedule  
+(8.0) Project team  
+(9.0) Glossary of terms  
+
+## (1.0) Product Overview
+
+We the people of “Atlantis” have resurfaced from the depths of the ocean. We have our own currency and bank but need an online banking system to reconnect with the world. The issues we have are the following:
+
+- Unable to e-transfer other banking accounts
+- Unable to view avaible funds
+- No online payment methods
+- The city of Atlantis is growing and visiting the bank is not scalable. Customers having to go to the bank just to view the transaction history is causing problems. 
+
+We predict that making an online banking system will solve these issues and increase customer satisfaction for all their banking needs. 
+
+## (2.0) Project Objectives
+
+The (name of bank) online banking system needs to fulfill the following needs
+
+- All users can view funds and transaction history, including but not limited too: withdrawls, purchases, and deposits. 
+- Users can view their credit card information, including but not limited too, credit card number, security code and credit limits.
+- Users can create investment and savings accounts
+- The sytem should be available 24/7
+- The system must be compatible with the Chrome browser as it is the only browser avaible to Atlanteans.
+- The system must supports up-to-date security measures
+
+## (3.0) Current System(s)
+
+The (name of bank) has a physcial bank and no online presence.
+
+## (4.0) Intended users of the system
+
+There are two intended users of the sytem
+- Bank Managers
+- Users of the bank
+
+ Users of the bank are the primary users and will be able to manage all of their banking needs through the chrome web browser. Users should be able to interact with the software through their personal computers or on mobile phones.
```

Comment:
I think we could replace "users" with "clients" here, since "users" of the system are both clients and managers, while "clients" is more specific like the people who open accounts and use the bank's services as customers. What do you guys think?
```suggestion
- Bank clients

 Clients of the bank are the primary users and will be able to manage all of their banking needs through the chrome web browser. Clients should be able to interact with the software through their personal computers or on mobile phones.
```